crucible-llvm/src/Lang/Crucible/LLVM/Translation/Aliases.hs

```diff
@@ -0,0 +1,150 @@
+-----------------------------------------------------------------------
```

Comment:
I assume all the code in this module was just moved verbatim?

Reply:
Mostly... but I think I changed the maps returned by `functionAliases` and `globalAliases` to be keyed just on the `Symbol` rather than on the `Global` or `Definition`.